features/step_definitions/branch_steps.rb

```diff
@@ -80,7 +80,16 @@
 end
 
 
+Given(/^I am on the "(.+?)" branch with "(.+?)" as the previous Git branch/) do |current_branch, previous_branch|
+  step %(I am on the "#{previous_branch}" branch)
+  step %(I am on the "#{current_branch}" branch)
```

Comment:
That's also true. Thinking about this more, I'm siding with @charlierudolph here.